docs/topics/ref_openjdk-images-for-red-hat-enterprise-linux.adoc

```diff
@@ -0,0 +1,30 @@
+[id='openjdk-images-for-red-hat-enterprise-linux_{context}']
+= OpenJDK images for Red Hat Enterprise Linux
+You can select the OpenJDK image used to build and deploy your application to OpenShift. 
+OpenJDK images are available for RHEL{nbsp}7 and RHEL{nbsp}8 with OpenJDK 8 or OpenJDK 11.
+
+[cols="1,1,1", options=header]
+|===
+|OS
+|JDK
+|Red Hat Container Catalog
+
+|RHEL 7
+|OpenJDK 8
+|link:https://access.redhat.com/containers/#/registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift[RHEL 7 with OpenJDK 8^]
+
+| RHEL 7
+|OpenJDK 11
+|link:https://access.redhat.com/containers/#/registry.access.redhat.com/openjdk/openjdk-11-rhel7[RHEL 7 with OpenJDK 11^]
+
+|RHEL 8
```

Comment:
The rhel8 images are accessible only with authentication. Please make a note about it.